release/release-srpm

```diff
@@ -187,6 +187,13 @@ prepare()
     printf "%%changelog\n" >> $logfile
     changelog_lines $TAG $RELEASE >> $logfile
 
+    # copy additional files available in source dir
+    if test -d "$SOURCE";then
+	    find $SOURCE -maxdepth 1 -type f | while read file; do
+	        cp "$file" $workdir
```

Comment:
Does this mean that the line below to copy the patch file is now superfluous? If so, could you adapt the following lines so that the patches are enumerated from $workdir instead?